.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.8.2
+    rev: v0.8.6
```

Comment:
ruff 0.9.2 is out by now, but not so important for this release

Reply:
I would leave this for `3.2.0`